packages/core/src/App/Containers/AccountTypesModal/account-types-modal.jsx

```diff
@@ -31,8 +32,14 @@ const Box = ({ title, description, footer_text, icons, cards }) => {
         <div className='account-types__box'>
             <div className='account-types__box-left'>
                 <h2 className='account-types__box-title'>{title}</h2>
-                <p className='account-types__box-description'>{description}</p>
-                {footer_text && <p className='account-types__box-footer'>{footer_text}</p>}
+                <Text as='p' size='xs' lineHeight='x' className='account-types__box-description'>
+                    {description}
+                </Text>
+                {footer_text && (
+                    <Text as='p' size='xxs' weight='bold' className='account-types__box-footer'>
+                        {footer_text}
+                    </Text>
+                )}
```

Comment:
Is it possible to change `Text` so it can internally check if children exist? I mean encapsulating `children && ...`.

Reply:
Since we are checking children for the other components outside of the component, and since I am done with 2 packages and in case of adding this prop, I should come back to them, I think we can keep this changing for the second phase if you agree.